instructions-for-devs.md

```diff
@@ -0,0 +1,203 @@
+# Instructions for Developers & Testers
+
+This document contains `xud` setup instructions for developers and everyone with a need for more fine-grained control when running `xud`.
+
+## 1. Install
+
+First, clone the repository from GitHub and install dependencies. 
+
+Developers & Testers:
+```bash
+git clone https://github.com/ExchangeUnion/xud
+cd xud
+npm install
+```
+For everyone else, we recommend using the [latest release](https://github.com/ExchangeUnion/xud/releases), below an example using the `1.0.0-prealpha` release tag:
+```bash
+git clone https://github.com/ExchangeUnion/xud 
+cd xud
+git checkout tags/1.0.0-prealpha -b 1.0.0-prealpha
+npm install
+```
+
+
+`xud` uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/). You will have to install one of those and create a user *xud* and a database *xud* and grant all permissions for the new database to the *xud* user.
+
+## 2. Prepare your environment 
+
+To initialize the database with some testing data, run the following command from `~/xud`:
```

Comment:
Lets not assume that the users will always clone xud into their home directory.

Reply:
Removed the ~, even though I think it's ok for instructional purposes. We can assume ppl who don't install in ~ , know what ~ is :)